train.py

```diff
@@ -291,11 +297,11 @@ def train():
                 if changed:
                     cfg.delayed_settings = [x for x in cfg.delayed_settings if x[0] > iteration]
 
-                # Warm up by linearly interpolating the learning rate from some smaller value
+                # Warm up by linearly interpolating the learning rate from some smaller value 
                 if cfg.lr_warmup_until > 0 and iteration <= cfg.lr_warmup_until:
```

Comment:
can we leave this fine tuning, lr management to the optimizer (or other code provided by the framework)?